randovania/games/prime2/game_data.py

```diff
@@ -111,6 +111,19 @@ def _exporter():
             "The energy gate that disappears after the pirate fight in Torvus Temple blocks this door.",
         ),
     ],
+    web_info=game.GameWebInfo(
+        what_can_randomize=[
+            "All items including Temple Keys",
+            "Elevator destinations",
+            "Starting locations",
+            "Door locks",
+            "Translator gate requirements",
+        ],
+        need_to_play=[
+            "An ISO of any Gamecube release of the game",
```

Comment:
NTSC and PAL regions only. No Dark Echoes allowed.

Reply:
Huh, i thought JP was also allowed.